STYLE-GUIDE.md

```diff
@@ -19,10 +19,28 @@ Words are important.  Pulumi strives to use language that is clear, harmonious,
 * Ensure that readers are able to scan the headings of a page and get an effective overview of the page's content.
 * Every page should have exactly one `h1`.
 * Headings levels should only increment one level at a time.  E.g., if your previous heading level was an `h2`, the next heading must be an `h2` or an `h3`, but not, e.g., an `h4` or `h5`.
+*  Include keywords relating to the content immediately following the heading in `h2` and deeper.
+*  Try to front-load your keywords into the sentence to more immediately indicate the subject matter of the following content.
```

Comment:
I need to gently push back against this suggestion. First and foremost, the goal of a post is to be readable, informative, and relatable; otherwise, people will stop reading our content and recommending it to others. Getting to that goal includes varying sentence structure, keeping the sentence structure engaging, and otherwise connecting with the reader. Sentences should first inform, then consider readability, and only then consider keywords.

Also, some edits:
```suggestion
*  Try to front-load your keywords into the sentence to indicate more immediately to the reader the subject matter of the following content.
```

Reply:
I agree, I held back from adding more nuance such as approaching the topic from differing perspectives in your writing to speak more specifically to different users, one such set of perspectives would be beginners, experienced users, and advanced professionals. Thank you for the wording update. In all things search, human spoken language far outweighs the use of keywords, they just set the tone.